llvm/include/llvm/ProfileData/MemProf.h

```diff
@@ -401,16 +424,21 @@ struct IndexedMemProfRecord {
       if (CallSites[I] != Other.CallSites[I])
         return false;
     }
+
+    if (Other.CallSiteIds != CallSiteIds)
```

Comment:
Ditto to same question in IndexedAllocationInfo operator==: can the CallSites based comparison above be removed?

Reply:
Yes, we can remove the `CallSites` comparison.  I was planning to remove it when we remove the `CallSites` field, but we can do so now.